store/src/main/java/org/apache/rocketmq/store/MappedFileQueue.java

```diff
@@ -191,7 +199,7 @@ public long howMuchFallBehind() {
         return 0;
     }
 
-    public MappedFile getLastMappedFile(final long startOffset, boolean needCreate) {
+    private MappedFile getLastMappedFile(final long startOffset, boolean needCreate) {
```

Comment:
I see, you need it to keep public as other `getLastMappedFile(...)` methods. Thanks for pointing to it!